templates/launch.html

```diff
@@ -238,10 +308,11 @@ <h3>${survey_type} Survey Metadata</h3>
     }
 
     function loadSurveyMetadata(schema_name, survey_type) {
-        document.querySelector("#submit-btn").disabled = true;
-        document.querySelector("#flush-btn").disabled = true;
+        document.querySelector("#submit-btn-v1").style.display = 'none';
+        document.querySelector("#submit-btn-v2").style.display = 'none';
+        document.querySelector("#flush-btn").style.display = 'none';
```

Comment:
Can reduce this to one line
```suggestion
        document.querySelectorAll("#submit-btn-v1", "#submit-btn-v2", "#flush-btn").style.display = 'none';
```